package.json

```diff
@@ -1,27 +1,32 @@
 {
     "name": "uriparser",
-    "version": "2.2.1",
+    "version": "2.3.0",
     "description": "Fastest uriparser, written in C",
-    "keywords": ["uriparser", "uri", "parser", "url"],
+    "keywords": [
+        "uriparser",
+        "uri",
+        "parser",
+        "url"
+    ],
     "contributors": [
         "Jakub Lekstan <kuebzky@gmail.com>",
         "Marcin Kaciuba <marcin.kaciuba@gmail.com>"
     ],
     "dependencies": {
-        "nan": "~2.9.2"
+        "nan": "^2.9.2"
     },
     "devDependencies": {
-      "jasmine-node": "1.14.5"
+        "jasmine-node": "1.14.5"
     },
     "scripts": {
-      "test": "jasmine-node tests/*"
+        "test": "jasmine-node tests/*"
     },
     "main": "./bin/uriparser.node",
     "repository": {
         "type": "git",
         "url": "https://github.com/DreamLab/uriparser.git"
     },
     "engines": {
-        "node": ">= 0.8.0"
+        "node": ">= 8.0.0"
```

Comment:
is it necessary?

Reply:
No. But this is only suggestion for npm. It will still work on node 0.8.
```
Unless the user has set the engine-strict config flag, this field is advisory only and will only produce warnings when your package is installed as a dependency.
```
https://docs.npmjs.com/files/package.json#engines

But if you want I can revert this.

Reply:
c'mon guys - we have 2018 ;-)